lib/src/play_store_search_api.dart

```diff
@@ -2,22 +2,16 @@
  * Copyright (c) 2021 William Kwabla. All rights reserved.
  */
 
+import 'package:html/dom.dart';
 import 'package:html/parser.dart' show parse;
 import 'package:http/http.dart' as http;
 
-class PlayStroeSearchApi {
+class PlayStroeSearchAPI {
```

Comment:
The spelling of this class should be `PlayStoreSearchAPI`.

Reply:
I will fix it ASAP